EXAMPLES.md

```diff
@@ -0,0 +1,473 @@
+test:
+	python main.py --bits 12 --max-codes 3 --width 500 --height 500 \
+		--output-dir ./test-targets
+	@echo "Test targets generated in ./test-targets/"
+
+# Calibration targets
+calibration:
+	python main.py --bits 12 --max-codes 25 --width 2000 --height 2000 \
+		--output-dir ./calibration-targets
+	@echo "Calibration targets generated in ./calibration-targets/"
+
+# Production targets  
+production:
+	python main.py --bits 12 --width 3000 --height 3000 \
+		--output-dir ./production-targets
+	@echo "Production targets generated in ./production-targets/"
+
+# Specialized targets with transitions
+filtered:
+	python main.py --bits 12 --transitions 3 --max-codes 20 \
+		--width 2500 --height 2500 --output-dir ./filtered-targets
+	@echo "Filtered targets generated in ./filtered-targets/"
+
+# Clean generated files
+clean:
+	rm -rf ./test-targets ./calibration-targets ./production-targets ./filtered-targets
+	@echo "Cleaned all generated target directories"
+
+# Validate setup
+validate:
+	python -c "import main; print('Import successful')"
+	python main.py --help > /dev/null
+	@echo "Setup validation passed"
+```
+
+### Integration with Shell Scripts
+
+```bash
+#!/bin/bash
+# generate_target_suite.sh - Generate complete target suite
+
+set -e  # Exit on any error
+
+echo "=== scomp-link Target Generation Suite ==="
+
+# Configuration
+BASE_DIR="./photogrammetry-targets"
+DATE=$(date +%Y%m%d)
+SUITE_DIR="${BASE_DIR}/${DATE}"
+
+# Create directory structure
+mkdir -p "${SUITE_DIR}"/{test,calibration,production,filtered}
+
+echo "Generating target suite in: ${SUITE_DIR}"
+
+# Generate test targets (fast)
+echo "1/4 Generating test targets..."
+python main.py --bits 12 --max-codes 5 --width 500 --height 500 \
+    --output-dir "${SUITE_DIR}/test"
+
+# Generate calibration targets
+echo "2/4 Generating calibration targets..."
+python main.py --bits 12 --max-codes 25 --width 2000 --height 2000 \
+    --output-dir "${SUITE_DIR}/calibration"
+
+# Generate filtered targets
+echo "3/4 Generating filtered targets..."
+python main.py --bits 12 --transitions 3 --max-codes 15 \
+    --width 2500 --height 2500 \
+    --output-dir "${SUITE_DIR}/filtered"
+
+# Generate production targets (slow)
+echo "4/4 Generating production targets (this may take several minutes)..."
+python main.py --bits 12 --width 3000 --height 3000 \
+    --output-dir "${SUITE_DIR}/production"
+
+# Generate summary
+echo ""
+echo "=== Generation Complete ==="
+echo "Target suite generated in: ${SUITE_DIR}"
+echo "Summary:"
+echo "  Test targets:       $(ls "${SUITE_DIR}/test" | wc -l)"
+echo "  Calibration targets: $(ls "${SUITE_DIR}/calibration" | wc -l)"
+echo "  Filtered targets:    $(ls "${SUITE_DIR}/filtered" | wc -l)"
+echo "  Production targets:  $(ls "${SUITE_DIR}/production" | wc -l)"
+
+# Calculate total size
+total_size=$(du -sh "${SUITE_DIR}" | cut -f1)
+echo "  Total size:         ${total_size}"
+
+echo ""
+echo "Ready for photogrammetry applications!"
+```
```

Comment:
Because we don't really have an API and haven't set this up as a package, Integration really just boils down to some fairly rudimentary scripting concepts. I don't think we need to cover it in our docs.